BritneyS/progress-bar
# Build progress bar

## What does this PR do :question:
This PR is for the completion of the progress bar, built with React hooks and styled components.


## How to test it :microscope:
1. Clone this branch and run `npm i && npm start`.
2. Navigate to http://localhost:3000
3. Click the button! The progress bar should grow until the button says `100` and no more.


## Any background info you would like to include :white_check_mark:
N/A


## Screenshots (if applicable) :camera:
![progress-bar-pr](https://user-images.githubusercontent.com/8409475/67128337-5d724080-f1c9-11e9-986b-ced3cb6b4520.gif)
